src/wiki/getting-started/installing-polymc.md

```diff
@@ -57,6 +57,12 @@ Depending on your system, you may need to grant the PolyMC AppImage **executable
 You can do this by opening your system's terminal application, **making sure to navigate to the location of the downloaded AppImage,** before granting the execute permission using this command:
 
 ```
-sudo chmod +x PolyMC-Linux-1.1.0-x86_64.appimage
+sudo chmod +x PolyMC-Linux-1.2.0-x86_64.appimage
 ```
 **Please note,** that depending on the version of PolyMC that you have downloaded, you may have to **change the version number** in the command above.
+
+
+### I want to make my system install portable 
+
+Since PolyMC 1.2.0, you can make any install portable (or making portable installs system) just by adding (or removing) [this file](https://raw.githubusercontent.com/PolyMC/PolyMC/develop/program_info/portable.txt) to the PolyMC root directory.
```

Comment:
just by creating or removing should be enough, copying the text is not needed

Reply:
oh